app/common/src/fileFilter.ts

```diff
@@ -0,0 +1,5 @@
+/** Describes allowed file types in the file browser. */
+export interface FileFilter {
+  name: string
+  extensions: 'all' | string[]
+}
```

Comment:
We could put that in GUI package. The GUI is devDependency of electron client anyway.

Reply:
Done